### PR TITLE
rebuild master for new toc

### LINKED ISSUE #1073: Getting started documentation link 404s on master
**What steps did you take and what happened:**
Go to https://heptio.github.io/ark/master/ and click the Getting started link in the left hand nav.

Page 404s https://heptio.github.io/ark/master/get-started

**What did you expect to happen:**

Page displays.

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

(also fixes a small weirdness with toc link to restic topic)

### CODE REVIEW

File: master/storage-layout-reorg-v0.10.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ## Overview
 
+<<<<<<< HEAD
```
Comment (skriss) [inline master/storage-layout-reorg-v0.10.md]: resolve

File: master/upgrading-to-v0.10.md [hunk 2]
```diff
@@ -19,13 +19,19 @@ some miscellaneous Ark settings. In v0.10, we've eliminated this CRD and replace
 When upgrading to v0.10, you'll need to transfer the configuration information that you currently have in the `Config` CRD
 into the above. We'll cover exactly how to do this below.
 
+<<<<<<< HEAD
```
Comment (skriss) [inline master/upgrading-to-v0.10.md]: resolve

File: master/upgrading-to-v0.10.md [hunk 3]
```diff
@@ -19,13 +19,19 @@ some miscellaneous Ark settings. In v0.10, we've eliminated this CRD and replace
...
+
+>>>>>>> rebuild master for new toc
 ### Reorganization of data in object storage
 
 We've made [changes to the layout of data stored in object storage][3] for simplicity and extensibility. You'll need to
 rearrange any pre-v0.10 data as part of the upgrade. We've provided a script to help with this.
 
 ## Step-by-Step Upgrade Instructions
 
+<<<<<<< HEAD
```
Comment (skriss) [inline master/upgrading-to-v0.10.md]: resolve

File: master/upgrading-to-v0.10.md [hunk 4]
```diff
@@ -66,6 +93,7 @@ the existing `Config` resource as necessary.
     `--restic-timeout` | 1h | how long backups/restores of pod volumes should be allowed to run before timing out (previously `podVolumeOperationTimeout` in the `Config` resource in pre-v0.10 versions)
     `--restore-only` | false | run in a mode where only restores are allowed; backups, schedules, and garbage-collection are all disabled
 
+<<<<<<< HEAD
```
Comment (skriss) [inline master/upgrading-to-v0.10.md]: resolve

File: master/versions.md [hunk 5]
```diff
@@ -14,5 +14,18 @@ Breaking changes are documented in the release notes and in the documentation.
 
 - See [Upgrading to version 0.10.0][2]
 
+<<<<<<< HEAD
```
Comment (skriss) [inline master/versions.md]: resolve

**Comment by Bradamant3** [comment]
rebuilt bc those strays were ... mysterious and I didn't trust my judgement on 'em. n.b. I started also to fix a problem with navigation and the need to add redirects, but it needs more attention than I can give it atm; filing an issue so we don't lose it

File: v0.9.0/quickstart.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+---
```
Comment (skriss) [inline v0.9.0/quickstart.md]: what's this for?  it's in the v0.9.0 dir?

Comment (nrb) [reply v0.9.0/quickstart.md]: I think it's for #1126; switching versions while on a renamed page will result in a 404.

**Comment by Bradamant3** [comment]
@skriss do you want me to remove the start at redirects? Or is it OK to leave in since it's not breaking anything/doesn't show up/is a start at a fix to issue 1073?

**Comment by skriss** [comment]
I'd rather remove it and just make sure the relevant issue is up-to-date (and could include this snippet and any notes about why it didn't work), if that's not too much effort.

**Comment by Bradamant3** [comment]
Useless snippet removed.

**Comment by skriss** [comment]
LGTM - if ready, pls squash and I'll merge.

**Comment by Bradamant3** [comment]
whoops, nope, need to write a new TOC now that 1098 is merged lololol. Incoming shortly ...

**Comment by Bradamant3** [comment]
New master docs build here: https://bradamant3.github.io/ark/master/get-started.html (that is, you can see the changes in this topic and its links)

**Comment by skriss** [comment]
LGTM - ready to merge?

**Pull request merged**